pos_order_reorder/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Point of Sale Re-order",
+    "version": "16.0.0.1.0",
+    "category": "Sales/Point of Sale",
+    "summary": "Simple Re-order in the Point of Sale ",
+    "depends": ["point_of_sale"],
+    "website": "https://github.com/OCA/pos",
+    "author": "Cetmix,Odoo Community Association (OCA)",
+    "data": [
+        "views/res_config_settings_view.xml",
+    ],
+    "installable": True,
+    "assets": {
+        "point_of_sale.assets": [
+            "pos_discount/static/src/js/**/*",
+            "pos_discount/static/src/xml/**/*",
```

Comment:
If I read correctly, the test error come from here. Not sure why we reference pos_discount here?